voxblox/include/voxblox/integrator/tsdf_integrator.h

```diff
@@ -83,15 +83,15 @@ class TsdfIntegratorBase {
   // mutex allowing it to grow during integration.
   // These temporary blocks can be merged into the layer later by calling
   // updateLayerWithStoredBlocks()
-  inline TsdfVoxel* allocateStorageAndGetVoxelPtr(
+  TsdfVoxel* allocateStorageAndGetVoxelPtr(
```

Comment:
This function and updateTsdfVoxel are probably the two most called functions in voxblox. Though I suspect that even without the inline specification that the compiler would usually inline them anyway. 

Maybe just check the timings before and after the change but I am guessing this won't make any difference to run time.

Reply:
I checked the timings of this branch vs current voxblox master using the voxblox_ros cow_and_lady_dataset.launch and can't see any difference between the run times of the two versions.

Reply:
@margaritaG thanks, will merge this then :)